src/server.rs

```diff
@@ -424,6 +613,18 @@ fn main() {
                     // Server tick
                     // Likely spawn off work to handle server tasks here
                     server_state.tick += 1;
+
+                    server_state.expire_old_messages();
```

Comment:
If performance is an issue, we should consider running this only every 100 ticks or so.

Reply:
Agreed. Let's wait and see.